### PR TITLE
Placeholders and switch to push dhcp options

### PR DESCRIPTION
**Pull request by stephdl**
https://github.com/NethServer/dev/issues/6146

### CODE REVIEW

File: ui/src/views/OpenVPNRW.vue [hunk 1]
```diff
@@ -658,64 +658,88 @@
...
+                    class="help-block"
+                  >{{$t('validation.validation_failed')}}: {{$t('validation.'+newConfiguration.errors.PushDhcpOptionsStatus.message)}}</span>
+                </div>
+              </div>
+              <div
+                v-show="newConfiguration.advanced && newConfiguration.PushDhcpOptionsStatus"
                 :class="['form-group', newConfiguration.errors.PushDomain.hasError ? 'has-error' : '']"
               >
                 <label
                   class="col-sm-3 control-label"
```
Comment (edospadoni) [inline ui/src/views/OpenVPNRW.vue]: To have correctly aligned items on the form we have to move `class="col-sm-3 control-label"` to `class="col-sm-5 control-label"`

File: ui/src/views/OpenVPNRW.vue [hunk 2]
```diff
@@ -658,64 +658,88 @@
...
                   for="textInput-modal-markup"
                 >{{$t('openvpn_rw.dhcp_domain')}}</label>
                 <div class="col-sm-9">
```
Comment (edospadoni) [inline ui/src/views/OpenVPNRW.vue]: To have correctly aligned items on the form we have to move `class="col-sm-9"` to `class="col-sm-7"`

**Comment by edospadoni** [comment]
Modal items are not aligned with previous items.

align-cols

We have to move all `label` and `div` class

**Comment by stephdl** [comment]
fun this is what I see with chromium

Screenshot (12)

**Comment by stephdl** [comment]
Screenshot (14)
Screenshot (13)

**Comment by stephdl** [comment]
I use https://www.npmjs.com/package/ip to calculate the first IP when the initial setup or if you want to change the Network or the Netmask in routed mode

Please comment.

**Comment by edospadoni** [comment]
> I use https://www.npmjs.com/package/ip to calculate the first IP when the initial setup or if you want to change the Network or the Netmask in routed mode
> 
> Please comment.

Yeah, now it's work great, I like also the placeholdere messages

File: package-lock.json [hunk 3]
```diff
@@ -0,0 +1,11 @@
+{
```
Comment (edospadoni) [inline package-lock.json]: This file, on the root of the project I think is useless. Only inside the `ui` directory is necessary

Comment (stephdl) [reply package-lock.json]: done

**Review by edospadoni — APPROVED** [review]

**Review by edospadoni — APPROVED** [review]

**Pull request merged**